tests/e2e/playground/playground.config.js

```diff
@@ -6,7 +6,7 @@ module.exports = defineConfig({
   ...cypressConfig,
   e2e: {
     ...cypressConfig.e2e,
-    specPattern: 'tests/e2e/playground/specs',
+    specPattern: 'tests/e2e/**/*.spec.js',
```

Comment:
This updates the playground config so that you can also see the 'regular' tests without quitting and changing the config